pkg/operator/staticpod/controller/backingresource/backing_resource_controller.go

```diff
@@ -1,15 +1,13 @@
 package backingresource
 
 import (
-	"path/filepath"
+	"embed"
 
 	"github.com/openshift/library-go/pkg/assets"
-	"github.com/openshift/library-go/pkg/operator/staticpod/controller/backingresource/bindata"
 )
 
-const (
-	manifestDir = "pkg/operator/staticpod/controller/backingresource"
-)
+//go:embed manifests
+var fs embed.FS
```

Comment:
i think we should use the same naming everywhere
`assets` as an example from other files

Reply:
`assets` conflicts with imported package name. I feel this is a tiny source file and easy to follow as is.

Reply:
@sanchezl 
I see some updates below, but what about this comment?

Reply:
I think it is pretty clear as-is. It is declared and used within 10 line of code.